src/gov/nist/javax/sip/SipStackImpl.java

```diff
@@ -1510,22 +1512,25 @@ public SipStackImpl(Properties configurationProperties)
 		
 		String valveClassName = configurationProperties.getProperty("gov.nist.javax.sip.SIP_MESSAGE_VALVE", null);
```

Comment:
there is a comment for every property supported in the stack. This javadoc must be updated to reflect the new comma separated support. Look in the beginning of the class